projects/SDE Lighthouse 03 - Kontopantelis/template-sql/1.patients.template.sql

```diff
@@ -37,10 +45,8 @@ SELECT
 	"EthnicityLatest_Category" AS "EthnicityCategory",
 	"IMD_Decile" AS "IMD2019Decile1IsMostDeprived10IsLeastDeprived",
 	"FirstDementiaDate",
-	CAST("RegisteredDateOfDeath" AS DATE) AS "RegisteredDateOfDeath"
+	TO_CHAR(alive."DeathDate", 'YYYY-MM') AS "YearAndMonthOfDeath"
```

Comment:
this is probably fine but I use DATE_TRUNC(month, DeathDate) which keeps the result in DATE format. e.g. it would change '2013-01-23' to '2013-01-01'. So the new variable can be called DeathYearAndMonth.

Reply:
OK